apps/daimo-web/next.config.js

```diff
@@ -19,6 +19,11 @@ const nextConfig = {
       test: /\.md$/,
       type: "asset/source",
     });
+    config.module.rules.push({
+      test: /\.tsx?$/,
```

Comment:
next handles ts by default, curious where it broke

Reply:
it complained the following about `assert.ts` import from `@daimo/common` (which was getting imported due importing `note.ts` from `@daimo/common`)

```
Module parse failed: Unexpected token (1:32)
You may need an appropriate loader to handle this file type
```

Adding this here fixed it. I suppose next doesn't handle it if the import is in a file with `use client`?